code/modules/clothing/spacesuits/syndi.dm

```diff
@@ -135,21 +135,21 @@
 	item_state = "syndicate-black-red"
 
 //Black-red syndicate contract varient
-/obj/item/clothing/head/helmet/space/syndicate/contract/black/red
+/obj/item/clothing/head/helmet/space/syndicate/contract
 	name = "contractor helmet"
-	desc = "A specalised black and red helmet that's quicker, and more compact that it's counterpart. Can be ultra-compressed into even the tightest of spaces."
+	desc = "A specalised black and gold helmet that's quicker, and more compact than it's standard counterpart. Can be ultra-compressed into even the tightest of spaces."

```

Comment:
"it's" shouldn't have an apostrophe in it.

Reply:
also "specialized"

Reply:
sorry I'm 2 IQ